### PR TITLE
Add events calendar

### PR DESCRIPTION
**Pull request by orkunkl**
@webmaster128 I just added this calendar plugin from a random website. Do you see any problems with using it?

### CODE REVIEW

File: resources/events.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<iframe id="open-web-calendar" 
+    style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+    src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fics.teamup.com%2Ffeed%2Fks3qwzicncjfkkeyq6%2F8703308.ics"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation"
+    allowTransparency="true" scrolling="no" 
+    frameborder="0" height="700px" width="120%"></iframe>
+
+You can subscribe to the events calendar using this [ICS feed link](https://ics.teamup.com/feed/ks3qwzicncjfkkeyq6/8703308.ics).
+
+Here are some articles explaining who to import ICS feeds:
```
Comment (webmaster128) [inline resources/events.md]:
```suggestion
Here are some articles explaining how to import ICS feeds:
```

File: resources/events.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+order: 1
+---
+
+# Events Calendar
+<iframe id="open-web-calendar" 
+    style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+    src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fics.teamup.com%2Ffeed%2Fks3qwzicncjfkkeyq6%2F8703308.ics"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation"
+    allowTransparency="true" scrolling="no" 
+    frameborder="0" height="700px" width="120%"></iframe>
```
Comment (webmaster128) [inline resources/events.md:12]: Why 120%? Does this work on mobile?

File: resources/events.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Events Calendar
+order: 1
+---
+
+# Events Calendar
+<iframe id="open-web-calendar" 
+    style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+    src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fics.teamup.com%2Ffeed%2Fks3qwzicncjfkkeyq6%2F8703308.ics"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation"
```
Comment (webmaster128) [inline resources/events.md:10]: A lot of permissions. Can we at least remove `allow-same-origin`?

**Review by webmaster128 — COMMENTED** [review]
Nice find. Can we change the color from blue to our purple?

Is the docs really the right place for this? Wat about moving it to the CosmWasm website?

> Do you see any problems with using it?

I'd be more relaxed when we host it on our own but it seems to do no harm

File: resources/events.md (hunk 2 above)
Comment (orkunkl) [reply resources/events.md:12]: Good question. Checking now.

File: resources/events.md (hunk 3 above)
Comment (orkunkl) [reply resources/events.md:10]:
```
Access to font at 'https://open-web-calendar.herokuapp.com/css/dhtmlx/dhtmlxscheduler_material/fonts.googleapis.com/KFOlCnqEu92Fr1MmEU9fBBc4.woff2' from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```
Got this error.

File: resources/events.md (hunk 2 above)
Comment (orkunkl) [reply resources/events.md:12]: Works but looks veeeryy weird
Screen Shot 2020-09-23 at 18 05 12

**Comment by orkunkl** [comment]
> Is the docs really the right place for this? Wat about moving it to the CosmWasm website?

@goodsurvivaltrack is adding to CosmWasm website as well.

**Comment by goodsurvivaltrack** [comment]
On CosmWasm website,  doc section directs to https://docs.cosmwasm.com/
I thought adding calendar here = adding calendar on cosmwasm website

Or Should I add calendar on cosmwasm website separately?

**Comment by webmaster128** [comment]
Do we really want dynamic content as such a calendar in our technical documentation (which I'd expect to work offline as well)? I'd rather add it to the website and don't add it here.

**Comment by orkunkl** [comment]
We don't use this calendar anymore, closing the PR

**Pull request closed without merging**